bluepysnap/spike_report.py

```diff
@@ -225,7 +225,7 @@ def log(self):
         path = Path(self.config["output_dir"]) / self.config["log_file"]
         if not path.exists():
             raise BluepySnapError("Cannot find the log file for the spike report.")
-        yield open(str(path), "r")
+        yield open(str(path), "r", encoding='utf-8')
```

Comment:
Unrelated to this PR, but it could use the path open method:
```suggestion
        yield path.open("r", encoding='utf-8')
```

Reply:
Cool. Might as well fix it.